modules/programs/taskwarrior.nix

```
@@ -88,7 +90,7 @@ in {
   config = mkIf cfg.enable {
     home.packages = [ pkgs.taskwarrior ];
 
-    xdg.configFile."task/taskrc".text = ''
+    xdg.configFile."task/taskrc_".text = ''
```

```suggestion
    home.file."${homeConf}".text = ''
```
This way the strings don't need to be kept in-sync.